integrationtests/osgi/src/test/resources/maven.properties

```diff
@@ -10,3 +10,4 @@ version.mockito=${version.mockito}
 version.mockito_dep.objenesis=${version.mockito_dep.objenesis}
 version.mockito_dep.bytebuddy=${version.mockito_dep.bytebuddy}
 localRepository=${settings.localRepository}
+version.hibernate.v53=${version.hibernate.v53}
```

Comment:
And this should be:

    version.hibernate.v53=${version.hibernate.core}

Reply:
I've removed `version.hibernate.v53` property in favor of `version.hibernate.osgi`